### DISCUSSION TITLE
Group by 24h period, from 8am to 8am

### DISCUSSION THREAD

**Discussion by AmegillaHouse** [kalkih/mini-graph-card#1152] [Q&A]
Is there a way that I can have this display data from a 24h period, starting 8am yesterday to 8am today? I want to use this as a rain meter to understand the rainfall over the last day, and currently it only shows me rainfall since midnight. 

This would be really helpful for me to know whether or not to water the garden.

**Comment by ildar170975** [comment]
It is not possible to set a particular time as “”end point”
